doc/security_considerations.md

```diff
@@ -1,15 +1,35 @@
-## Security considerations
+<!---
+Copyright 2018-2021 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--->
+
+# Security considerations
+
 During the architecture of Palisade we spent some time thinking about how we would be able to authenticate who the user is that is requesting access to the data.
```

Comment:
"During the architecture of Palisade" doesn't make sense. Maybe during the architecture "discussions" or "planning" would make more sense?